core-bundle/src/Resources/contao/pages/PageRegular.php

```diff
@@ -210,7 +210,7 @@ protected function prepare($objPage)
 		}
 
 		// Assign the title and description
-		$this->Template->title = StringUtil::stripInsertTags($this->replaceInsertTags($objLayout->titleTag)); // see #7097
+		$this->Template->title = $this->replaceInsertTags($objLayout->titleTag);
```

Comment:
I wonder if we should `strip_tags()` though? What if I'm using the `{{br}}` insert tag? Or whatever insert tag that generates HTML?

Reply:
We did not strip HTML tags before, but I guess it cannot hurt. Added in 31a24547dceb1ca23c1efe5595793535c70f29ee.